crates/stdarch-gen/Makefile

```diff
@@ -0,0 +1,5 @@
+all:
+	cargo run
+	rustfmt arm.rs aarch64.rs
```

Comment:
Why not run rustfmt directly from `main.rs`?

Reply:
`main.rs` isn't generated `arm.rs` and `aarch64.rs` both are so they're not human edited.

Reply:
I mean that `main.rs` will run rustfmt on the files it generated itself.

Reply:
oh I dind't think of that!